XIVSlothCombo/Combos/RPR.cs

```diff
@@ -161,11 +161,9 @@ protected override uint Invoke(uint actionID, uint lastComboMove, float comboTim
                     return OriginalHook(RPR.Gibbet);
                 }
 
-
-                //this seems not okay
                 if (IsEnabled(CustomComboPreset.ReaperGibbetGallowsFeatureOption) && (HasEffect(RPR.Buffs.SoulReaver) || HasEffect(RPR.Buffs.Enshrouded)))
                 {
-                    if ((HasEffect(RPR.Buffs.EnhancedGibbet) && !HasEffect(RPR.Buffs.Enshrouded) && IsEnabled(CustomComboPreset.ReaperGibbetGallowsFeatureOption)) || (HasEffect(RPR.Buffs.EnhancedCrossReaping) && HasEffect(RPR.Buffs.Enshrouded)))
+                    if ((HasEffect(RPR.Buffs.EnhancedGibbet) && !HasEffect(RPR.Buffs.Enshrouded) && IsEnabled(CustomComboPreset.ReaperGibbetGallowsFeatureOption)) || (HasEffect(RPR.Buffs.EnhancedVoidReaping) && HasEffect(RPR.Buffs.Enshrouded)))
```

Comment:
Duplicate call for IsEnabled(CustomComboPreset.ReaperGibbetGallowsFeatureOption))

Reply:
Cleaned up, these multiple calls are getting kind of annoying to clean up though.